kore/src/main/scala/org/kframework/attributes/Att.scala

```diff
@@ -170,96 +196,96 @@ object Att {
+  final val BINDER = Key("binder", KeyType.BuiltIn, KeyParameter.Optional)
+  final val BRACKET = Key("bracket", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val CELL = Key("cell", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val CELL_COLLECTION = Key("cellCollection", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val CELL_NAME = Key("cellName", KeyType.BuiltIn, KeyParameter.Required)
+  final val CIRCULARITY = Key("circularity", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val COLOR = Key("color", KeyType.BuiltIn, KeyParameter.Required)
+  final val COLORS = Key("colors", KeyType.BuiltIn, KeyParameter.Required)
+  final val COMM = Key("comm", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val CONCRETE = Key("concrete", KeyType.BuiltIn, KeyParameter.Optional)
+  final val CONSTRUCTOR = Key("constructor", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val CONTEXT = Key("context", KeyType.BuiltIn, KeyParameter.Required)
+  final val COOL = Key("cool", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val DEPENDS = Key("depends", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val ELEMENT = Key("element", KeyType.BuiltIn, KeyParameter.Required)
+  final val EXIT = Key("exit", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val FORMAT = Key("format", KeyType.BuiltIn, KeyParameter.Required)
+  final val FRESH_GENERATOR = Key("freshGenerator", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val FUNCTION = Key("function", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val FUNCTIONAL = Key("functional", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val GROUP = Key("group", KeyType.BuiltIn, KeyParameter.Required)
+  final val HASKELL = Key("haskell", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val HEAT = Key("heat", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val HYBRID = Key("hybrid", KeyType.BuiltIn, KeyParameter.Optional)
+  final val HOOK = Key("hook", KeyType.BuiltIn, KeyParameter.Required)
+  final val IDEM = Key("idem", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val IMPURE = Key("impure", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val INDEX = Key("index", KeyType.BuiltIn, KeyParameter.Required)
+  final val INITIAL = Key("initial", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val INITIALIZER = Key("initializer", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val INJECTIVE = Key("injective", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val INTERNAL = Key("internal", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val KAST = Key("kast", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val KLABEL = Key("klabel", KeyType.BuiltIn, KeyParameter.Required)
+  final val KORE = Key("kore", KeyType.BuiltIn, KeyParameter.Forbidden)
+  final val LABEL = Key("label", KeyType.BuiltIn, KeyParameter.Required)
+  final val LATEX = Key("latex", KeyType.BuiltIn, KeyParameter.Required)
+  final val LEFT = Key("left", KeyType.BuiltIn, KeyParameter.Optional)
```

Comment:
This one is forbidden, right?

Reply:
Thank you for bringing these up, I forgot about them. I set some of these to Optional because they were being generated by the compiler in a way that deviates from our discussions. For the left/right attributes, ModuleToKORE is making them with values:

https://github.com/runtimeverification/k/blob/a863a1e4618067b7027a12bcdbd2d1fca3ba8273/kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java#L1606-L1607